src/components/Upgrade/UpgradeOrganizationPanel.js

```diff
@@ -103,29 +103,15 @@ const UpgradeOrganizationPanel = React.memo(
         <Part>
           <Heading2>Source code</Heading2>
           <p>
-            {sourceUrl ? (
-              <SafeLink href={sourceUrl}>{sourceUrl}</SafeLink>
-            ) : (
-              'There is no available source for this app.'
-            )}
+            <SafeLink href={sourceUrl}>{sourceUrl}</SafeLink>
```

Comment:
Should we add `target="_blank"` to these?

Reply:
Yes, and that means we need to do it in the App center upgrade panel too :+1:

Reply:
Done, moved to `ExternalLink`.